.github/workflows/nightly-standalone-release.yml

```diff
@@ -1,4 +1,4 @@
-name: Nightly Test run
+name: Nightly Test run (Release)
```

Comment:
Can you rename this to "Jahia Release", just to clearly state that we're talking about the release of Jahia, while still testing the snapshot of the module.

You can also remove "Test run" to make it shorter.

`Nightly (Jahia Release)`

Reply:
Fixed